test/testMdastToVDOM.js

```diff
@@ -65,10 +65,10 @@ describe('Test MDAST to VDOM Transformation', () => {
   it('Programmatic Matcher Function', () => {
     const mdast = fs.readJSONSync(path.resolve(__dirname, 'fixtures', 'simple.json'));
     const transformer = new VDOM(mdast, action.secrets);
-    transformer.match(({ type }) => type === 'heading', () => '<h1>All Headings are the same to me</h1>');
+    transformer.match(({ type }) => type === 'heading', () => '<h1 id="h1">All Headings are the same to me</h1>');
```

Comment:
is this relevant to the XSS changes?

Reply:
It is not necessarily needed, but was an easy place to test the DOM clobbering escape. I could duplicate the test otherwise to cover that use case

Reply:
Removed as I now have dedicated tests for XSS and DOM Clobbering